specs/phase0/p2p-interface.md

```diff
@@ -567,6 +567,9 @@ The response code can have one of the following values, encoded as a single unsi
   The response payload adheres to the `ErrorMessage` schema (described below).
 -  2: **ServerError** -- the responder encountered an error while processing the request.
   The response payload adheres to the `ErrorMessage` schema (described below).
+-  3: **ResourceUnavailable** -- the responder does not have requested resource.
+  The response payload adheres to the `ErrorMessage` schema (described below).
+  *Note*: This response code is only valid as a response to `BlocksByRange`.
```

Comment:
There's a similar method in the sharding ReqResp spec, but that's unstable, and may be removed. Probably better for a later PR.

Reply:
sounds good